_i18n/pt-br.yml

```diff
@@ -43,47 +45,44 @@ explore:
-  question_fourteen: 14. O que é a Rosie?
-  answer_fourteen: A <a href="https://github.com/datasciencebr/rosie" target="_blank">Rosie</a> é a nossa Robô que foi programada para identificar usos ilegais da verba pública, começando pela CEAP. Ela julga cada um dos reembolsos solicitados pelos nossos deputados, nos dizendo quais são os motivos que fazem cada um poderem ser ilegais.
-  question_fifteen: 15. Como o projeto nasceu?
-  answer_fifteen: A <a href="/sobre" target="_blank">primeira fase</a> do projeto, que durou dois meses, foi realizada através de uma <a href="https://www.catarse.me/serenata" target="_blank">campanha de financiamento coletivo</a>. Mais de 1296 nos ajudaram a tornar o projeto realidade.
-  question_sixteen: 16. Como posso contribuir com o projeto?
-  answer_sixteen: Você pode contribuir para o projeto com código, ajudando a manter a equipe, com ideias, com <a href="https://twitter.com/search?q=%23serenatadeamor" target="_blank">retuítes</a> e compartilhamentos. Para entender como começar a contribuir com código, leia <a href="https://datasciencebr.com/how-does-one-contribute-to-serenata-de-amor-operation-36e3e7b38207#.een62m7or" target="_blank">este post</a>. Para contribuir diretamente para a continuação do projeto, você pode acessar a nossa página no <a href="https://apoia.se/serenata" target="_blank">apoia.se</a>.
-  question_seventeen: 17. O que o nome Serenata de Amor significa?
-  answer_seventeen: 'Os motivos são três:<br> 1 - Nos inspiramos diretamente no famoso Caso Toblerone - um famoso acontecimento onde uma política na Suécia renunciou o seu cargo após ser flagrada com um simples Toblerone na fatura de seu cartão corporativo. Queremos fazer isso: encontrar corrupção em pequenos gastos, mas em volume grande.<br> 2 - Parece nome de operação da Polícia Federal, e isso é muito legal.<br> 3 - É a nossa Serenata de Amor para o Brasil.'
-  question_eighteen: 18. Posso usar o código da Serenata de Amor na minha cidade, no meu estado ou no meu país?
-  answer_eighteen: Claro, essa ideia é sensacional — e é um dos principais motivos de adotarmos código aberto. <a href="https://medium.com/data-science-brigade/expandindo-a-serenata-de-amor-para-outras-esferas-48bed204d36d">Esse artigo explica isso pode ocorrer</a>.
+  question_one: 1. O que o nome Serenata de Amor significa?
+  answer_one: 'Os motivos são três:<br> 1 - Nos inspiramos diretamente no famoso Caso Toblerone - um famoso acontecimento onde uma política na Suécia renunciou o seu cargo após ser flagrada com um simples Toblerone na fatura de seu cartão corporativo. Queremos fazer isso: encontrar corrupção em pequenos gastos, mas em volume grande.<br> 2 - Parece nome de operação da Polícia Federal, e isso é muito legal.<br> 3 - É a nossa Serenata de Amor para o Brasil.'
```

Comment:
> Nos inspiramos diretamente no famoso Caso Toblerone - um famoso acontecimento […]

Is necessary this repetition of _famoso_? When reviewing the English version I removed it. I suggest we do the same in the Portuguese version.

Reply:
Yet I raise the question: we are not focusing in _corruption_ anymore, so I find this answer quite unfit for our present moment. I suggested something different in the English review.

Reply:
Completely agree, I was changing the questions that had that word to social control, but that one passed by me; Thanks @cuducos